src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationRecordPattern.java

```diff
@@ -0,0 +1,74 @@
+//non-compiled with javac: Compilable with Java21        //indent:0 exp:0
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;//indent:0 exp:0
+//indent:0 exp:0
+//indent:0 exp:0
+/* Config:                                                                  //indent:0 exp:0
+ * basicOffset = 4                                                          //indent:1 exp:1
+ * braceAdjustment = 0                                                      //indent:1 exp:1
+ * caseIndent = 4                                                           //indent:1 exp:1
+ * tabWidth = 4                                                             //indent:1 exp:1
+ * throwsIndent = 8                                                         //indent:1 exp:1
+ * forceStrictCondition = true                                             //indent:1 exp:1
```

Comment:
I mentioned this in other PR, please ensure you are consistent with comment indentation and placement.

Reply:
done